unity-renderer/Assets/Rendering/Culling/CullingController.cs

```diff
@@ -161,7 +161,7 @@ internal IEnumerator ProcessProfileWithEnabledCulling(CullingControllerProfile p
 
             foreach (Renderer r in renderers)
             {
-                if (r == null)
+                if (!r)
```

Comment:
please revert it. _**Don't omit comparison to null**_ is in our [code-guidlines](https://github.com/decentraland/unity-renderer/blob/dev/docs/style-guidelines.md#general)